pkg/lib/template/template_loader.go

```diff
@@ -151,6 +151,21 @@ func (loader *Loader) Render(
 			}
 			return data, err
 		},
+		"parse_csv": func(path string, delimiter rune) ([]map[string]interface{}, error) {
+			_, file, err := util.OpenFileOrUrl(path, rootDir)
+			if err != nil {
+				return nil, err
```

Comment:
Bitte auch Dokumentation in README.md für diese Funktion

Reply:
https://github.com/programmfabrik/apitest/pull/13/commits/4822986d507bcd0163e0a1ceb9232ec1100f05b9